### PR TITLE
Improve definition of Node's ChildProcess properties and "fork" function signature

### PR DESCRIPTION
**Pull request by lumaxis**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/dist/latest-v8.x/docs/api/child_process.html#child_process_child_process_fork_modulepath_args_options (and respective URLs for the other versions)
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/node/test/child_process.ts [hunk 1]
```diff
@@ -59,6 +59,20 @@ import { Writable, Readable, Pipe } from 'stream';
     ipc.ref();
 }
 
+{
+    const forked = childProcess.fork('./', {
+        windowsVerbatimArguments: true,
+        silent: false,
+        stdio: ["inherit"],
```
Comment (nguymin4) [inline types/node/test/child_process.ts:66]: A bit inconsistent double quote and single quote. But anyway, we don't have strict rule for this.

**Review by nguymin4 — APPROVED** [review]
LGTM!

File: types/node/index.d.ts [hunk 2]
```diff
@@ -22,7 +22,6 @@
 //                 Mariusz Wiktorczyk <https://github.com/mwiktorczyk>
 //                 Mohsen Azimi <https://github.com/mohsen1>
 //                 Nicolas Even <https://github.com/n-e>
-//                 Nicolas Voigt <https://github.com/octo-sniffle>
```
Comment (SimonSchick) [inline types/node/index.d.ts:25]: Why this change?

**Review by SimonSchick — CHANGES_REQUESTED** [review]

File: types/node/index.d.ts (hunk 2 above)
Comment (lumaxis) [reply types/node/index.d.ts:25]: @SimonSchick The account doesn't exist anymore. I noticed because it's not properly linked in @typescript-bot's comments.

Comment (SimonSchick) [reply types/node/index.d.ts:25]: Ok, in the future consider adding a note for unrelated changes such as these as it reduces roundtrip time.

**Review by SimonSchick — APPROVED** [review]

**Comment by lumaxis** [comment]
Ready to merge
